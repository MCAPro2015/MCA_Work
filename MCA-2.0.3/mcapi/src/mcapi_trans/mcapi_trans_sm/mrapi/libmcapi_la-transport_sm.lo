# libmcapi_la-transport_sm.lo - a libtool object file
# Generated by ltmain.sh (GNU libtool) 2.2.6
#
# Please DO NOT delete this file!
# It is necessary for linking the library.

# Name of the PIC object.
pic_object='.libs/libmcapi_la-transport_sm.o'

# Name of the non-PIC object
non_pic_object='libmcapi_la-transport_sm.o'

