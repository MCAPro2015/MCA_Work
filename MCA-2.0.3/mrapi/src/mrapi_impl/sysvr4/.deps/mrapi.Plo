mrapi.lo: ../../mrapi.c ../../../../mrapi/include/mrapi.h \
 /opt/freescale/usr/local/gcc-4.4.78-eglibc-2.10.78-1/powerpc-linux-gnu/bin/../powerpc-linux-gnu/libc/usr/include/assert.h \
 /opt/freescale/usr/local/gcc-4.4.78-eglibc-2.10.78-1/powerpc-linux-gnu/bin/../powerpc-linux-gnu/libc/usr/include/features.h \
 /opt/freescale/usr/local/gcc-4.4.78-eglibc-2.10.78-1/powerpc-linux-gnu/bin/../powerpc-linux-gnu/libc/usr/include/bits/predefs.h \
 /opt/freescale/usr/local/gcc-4.4.78-eglibc-2.10.78-1/powerpc-linux-gnu/bin/../powerpc-linux-gnu/libc/usr/include/sys/cdefs.h \
 /opt/freescale/usr/local/gcc-4.4.78-eglibc-2.10.78-1/powerpc-linux-gnu/bin/../powerpc-linux-gnu/libc/usr/include/bits/wordsize.h \
 /opt/freescale/usr/local/gcc-4.4.78-eglibc-2.10.78-1/powerpc-linux-gnu/bin/../powerpc-linux-gnu/libc/usr/include/gnu/stubs.h \
 /opt/freescale/usr/local/gcc-4.4.78-eglibc-2.10.78-1/powerpc-linux-gnu/bin/../powerpc-linux-gnu/libc/usr/include/gnu/stubs-32.h \
 /opt/freescale/usr/local/gcc-4.4.78-eglibc-2.10.78-1/powerpc-linux-gnu/bin/../lib/gcc/powerpc-linux-gnu/4.4.1/include/stddef.h \
 ../../../../mca_config.h \
 /opt/freescale/usr/local/gcc-4.4.78-eglibc-2.10.78-1/powerpc-linux-gnu/bin/../powerpc-linux-gnu/libc/usr/include/inttypes.h \
 /opt/freescale/usr/local/gcc-4.4.78-eglibc-2.10.78-1/powerpc-linux-gnu/bin/../powerpc-linux-gnu/libc/usr/include/stdint.h \
 /opt/freescale/usr/local/gcc-4.4.78-eglibc-2.10.78-1/powerpc-linux-gnu/bin/../powerpc-linux-gnu/libc/usr/include/bits/wchar.h \
 ../../../../mca.h ../../../../mca_impl_spec.h \
 ../../../../mrapi/src/mrapi_impl/mrapi_impl_spec.h \
 /opt/freescale/usr/local/gcc-4.4.78-eglibc-2.10.78-1/powerpc-linux-gnu/bin/../powerpc-linux-gnu/libc/usr/include/signal.h \
 /opt/freescale/usr/local/gcc-4.4.78-eglibc-2.10.78-1/powerpc-linux-gnu/bin/../powerpc-linux-gnu/libc/usr/include/bits/sigset.h \
 /opt/freescale/usr/local/gcc-4.4.78-eglibc-2.10.78-1/powerpc-linux-gnu/bin/../powerpc-linux-gnu/libc/usr/include/bits/types.h \
 /opt/freescale/usr/local/gcc-4.4.78-eglibc-2.10.78-1/powerpc-linux-gnu/bin/../powerpc-linux-gnu/libc/usr/include/bits/typesizes.h \
 /opt/freescale/usr/local/gcc-4.4.78-eglibc-2.10.78-1/powerpc-linux-gnu/bin/../powerpc-linux-gnu/libc/usr/include/bits/signum.h \
 /opt/freescale/usr/local/gcc-4.4.78-eglibc-2.10.78-1/powerpc-linux-gnu/bin/../powerpc-linux-gnu/libc/usr/include/time.h \
 /opt/freescale/usr/local/gcc-4.4.78-eglibc-2.10.78-1/powerpc-linux-gnu/bin/../powerpc-linux-gnu/libc/usr/include/bits/siginfo.h \
 /opt/freescale/usr/local/gcc-4.4.78-eglibc-2.10.78-1/powerpc-linux-gnu/bin/../powerpc-linux-gnu/libc/usr/include/bits/sigaction.h \
 /opt/freescale/usr/local/gcc-4.4.78-eglibc-2.10.78-1/powerpc-linux-gnu/bin/../powerpc-linux-gnu/libc/usr/include/bits/sigcontext.h \
 /opt/freescale/usr/local/gcc-4.4.78-eglibc-2.10.78-1/powerpc-linux-gnu/bin/../powerpc-linux-gnu/libc/usr/include/asm/sigcontext.h \
 /opt/freescale/usr/local/gcc-4.4.78-eglibc-2.10.78-1/powerpc-linux-gnu/bin/../powerpc-linux-gnu/libc/usr/include/asm/ptrace.h \
 /opt/freescale/usr/local/gcc-4.4.78-eglibc-2.10.78-1/powerpc-linux-gnu/bin/../powerpc-linux-gnu/libc/usr/include/bits/sigstack.h \
 /opt/freescale/usr/local/gcc-4.4.78-eglibc-2.10.78-1/powerpc-linux-gnu/bin/../powerpc-linux-gnu/libc/usr/include/bits/pthreadtypes.h \
 /opt/freescale/usr/local/gcc-4.4.78-eglibc-2.10.78-1/powerpc-linux-gnu/bin/../powerpc-linux-gnu/libc/usr/include/bits/sigthread.h \
 /opt/freescale/usr/local/gcc-4.4.78-eglibc-2.10.78-1/powerpc-linux-gnu/bin/../powerpc-linux-gnu/libc/usr/include/pthread.h \
 /opt/freescale/usr/local/gcc-4.4.78-eglibc-2.10.78-1/powerpc-linux-gnu/bin/../powerpc-linux-gnu/libc/usr/include/endian.h \
 /opt/freescale/usr/local/gcc-4.4.78-eglibc-2.10.78-1/powerpc-linux-gnu/bin/../powerpc-linux-gnu/libc/usr/include/bits/endian.h \
 /opt/freescale/usr/local/gcc-4.4.78-eglibc-2.10.78-1/powerpc-linux-gnu/bin/../powerpc-linux-gnu/libc/usr/include/bits/byteswap.h \
 /opt/freescale/usr/local/gcc-4.4.78-eglibc-2.10.78-1/powerpc-linux-gnu/bin/../powerpc-linux-gnu/libc/usr/include/sched.h \
 /opt/freescale/usr/local/gcc-4.4.78-eglibc-2.10.78-1/powerpc-linux-gnu/bin/../powerpc-linux-gnu/libc/usr/include/bits/sched.h \
 /opt/freescale/usr/local/gcc-4.4.78-eglibc-2.10.78-1/powerpc-linux-gnu/bin/../powerpc-linux-gnu/libc/usr/include/bits/time.h \
 /opt/freescale/usr/local/gcc-4.4.78-eglibc-2.10.78-1/powerpc-linux-gnu/bin/../powerpc-linux-gnu/libc/usr/include/xlocale.h \
 /opt/freescale/usr/local/gcc-4.4.78-eglibc-2.10.78-1/powerpc-linux-gnu/bin/../powerpc-linux-gnu/libc/usr/include/bits/setjmp.h \
 /opt/freescale/usr/local/gcc-4.4.78-eglibc-2.10.78-1/powerpc-linux-gnu/bin/../powerpc-linux-gnu/libc/usr/include/unistd.h \
 /opt/freescale/usr/local/gcc-4.4.78-eglibc-2.10.78-1/powerpc-linux-gnu/bin/../powerpc-linux-gnu/libc/usr/include/bits/posix_opt.h \
 /opt/freescale/usr/local/gcc-4.4.78-eglibc-2.10.78-1/powerpc-linux-gnu/bin/../powerpc-linux-gnu/libc/usr/include/bits/confname.h \
 /opt/freescale/usr/local/gcc-4.4.78-eglibc-2.10.78-1/powerpc-linux-gnu/bin/../powerpc-linux-gnu/libc/usr/include/getopt.h \
 /opt/freescale/usr/local/gcc-4.4.78-eglibc-2.10.78-1/powerpc-linux-gnu/bin/../powerpc-linux-gnu/libc/usr/include/sys/shm.h \
 /opt/freescale/usr/local/gcc-4.4.78-eglibc-2.10.78-1/powerpc-linux-gnu/bin/../powerpc-linux-gnu/libc/usr/include/sys/ipc.h \
 /opt/freescale/usr/local/gcc-4.4.78-eglibc-2.10.78-1/powerpc-linux-gnu/bin/../powerpc-linux-gnu/libc/usr/include/bits/ipctypes.h \
 /opt/freescale/usr/local/gcc-4.4.78-eglibc-2.10.78-1/powerpc-linux-gnu/bin/../powerpc-linux-gnu/libc/usr/include/bits/ipc.h \
 /opt/freescale/usr/local/gcc-4.4.78-eglibc-2.10.78-1/powerpc-linux-gnu/bin/../powerpc-linux-gnu/libc/usr/include/bits/shm.h \
 /opt/freescale/usr/local/gcc-4.4.78-eglibc-2.10.78-1/powerpc-linux-gnu/bin/../powerpc-linux-gnu/libc/usr/include/sys/sem.h \
 /opt/freescale/usr/local/gcc-4.4.78-eglibc-2.10.78-1/powerpc-linux-gnu/bin/../powerpc-linux-gnu/libc/usr/include/bits/sem.h \
 /opt/freescale/usr/local/gcc-4.4.78-eglibc-2.10.78-1/powerpc-linux-gnu/bin/../powerpc-linux-gnu/libc/usr/include/sys/types.h \
 /opt/freescale/usr/local/gcc-4.4.78-eglibc-2.10.78-1/powerpc-linux-gnu/bin/../powerpc-linux-gnu/libc/usr/include/sys/select.h \
 /opt/freescale/usr/local/gcc-4.4.78-eglibc-2.10.78-1/powerpc-linux-gnu/bin/../powerpc-linux-gnu/libc/usr/include/bits/select.h \
 /opt/freescale/usr/local/gcc-4.4.78-eglibc-2.10.78-1/powerpc-linux-gnu/bin/../powerpc-linux-gnu/libc/usr/include/sys/sysmacros.h \
 /opt/freescale/usr/local/gcc-4.4.78-eglibc-2.10.78-1/powerpc-linux-gnu/bin/../powerpc-linux-gnu/libc/usr/include/ctype.h \
 /opt/freescale/usr/local/gcc-4.4.78-eglibc-2.10.78-1/powerpc-linux-gnu/bin/../powerpc-linux-gnu/libc/usr/include/pwd.h \
 /opt/freescale/usr/local/gcc-4.4.78-eglibc-2.10.78-1/powerpc-linux-gnu/bin/../powerpc-linux-gnu/libc/usr/include/stdio.h \
 /opt/freescale/usr/local/gcc-4.4.78-eglibc-2.10.78-1/powerpc-linux-gnu/bin/../powerpc-linux-gnu/libc/usr/include/stdlib.h \
 /opt/freescale/usr/local/gcc-4.4.78-eglibc-2.10.78-1/powerpc-linux-gnu/bin/../powerpc-linux-gnu/libc/usr/include/alloca.h \
 /opt/freescale/usr/local/gcc-4.4.78-eglibc-2.10.78-1/powerpc-linux-gnu/bin/../powerpc-linux-gnu/libc/usr/include/string.h \
 /opt/freescale/usr/local/gcc-4.4.78-eglibc-2.10.78-1/powerpc-linux-gnu/bin/../powerpc-linux-gnu/libc/usr/include/bits/string.h \
 /opt/freescale/usr/local/gcc-4.4.78-eglibc-2.10.78-1/powerpc-linux-gnu/bin/../powerpc-linux-gnu/libc/usr/include/bits/string2.h \
 /opt/freescale/usr/local/gcc-4.4.78-eglibc-2.10.78-1/powerpc-linux-gnu/bin/../powerpc-linux-gnu/libc/usr/include/errno.h \
 /opt/freescale/usr/local/gcc-4.4.78-eglibc-2.10.78-1/powerpc-linux-gnu/bin/../powerpc-linux-gnu/libc/usr/include/bits/errno.h \
 /opt/freescale/usr/local/gcc-4.4.78-eglibc-2.10.78-1/powerpc-linux-gnu/bin/../powerpc-linux-gnu/libc/usr/include/linux/errno.h \
 /opt/freescale/usr/local/gcc-4.4.78-eglibc-2.10.78-1/powerpc-linux-gnu/bin/../powerpc-linux-gnu/libc/usr/include/asm/errno.h \
 /opt/freescale/usr/local/gcc-4.4.78-eglibc-2.10.78-1/powerpc-linux-gnu/bin/../powerpc-linux-gnu/libc/usr/include/asm-generic/errno.h \
 /opt/freescale/usr/local/gcc-4.4.78-eglibc-2.10.78-1/powerpc-linux-gnu/bin/../powerpc-linux-gnu/libc/usr/include/asm-generic/errno-base.h \
 ../../../../common/mca_utils.h \
 /opt/freescale/usr/local/gcc-4.4.78-eglibc-2.10.78-1/powerpc-linux-gnu/bin/../lib/gcc/powerpc-linux-gnu/4.4.1/include/stdarg.h \
 /opt/freescale/usr/local/gcc-4.4.78-eglibc-2.10.78-1/powerpc-linux-gnu/bin/../powerpc-linux-gnu/libc/usr/include/libio.h \
 /opt/freescale/usr/local/gcc-4.4.78-eglibc-2.10.78-1/powerpc-linux-gnu/bin/../powerpc-linux-gnu/libc/usr/include/_G_config.h \
 /opt/freescale/usr/local/gcc-4.4.78-eglibc-2.10.78-1/powerpc-linux-gnu/bin/../powerpc-linux-gnu/libc/usr/include/wchar.h \
 /opt/freescale/usr/local/gcc-4.4.78-eglibc-2.10.78-1/powerpc-linux-gnu/bin/../powerpc-linux-gnu/libc/usr/include/bits/stdio_lim.h \
 /opt/freescale/usr/local/gcc-4.4.78-eglibc-2.10.78-1/powerpc-linux-gnu/bin/../powerpc-linux-gnu/libc/usr/include/bits/sys_errlist.h \
 /opt/freescale/usr/local/gcc-4.4.78-eglibc-2.10.78-1/powerpc-linux-gnu/bin/../powerpc-linux-gnu/libc/usr/include/bits/stdio.h

../../../../mrapi/include/mrapi.h:

/opt/freescale/usr/local/gcc-4.4.78-eglibc-2.10.78-1/powerpc-linux-gnu/bin/../powerpc-linux-gnu/libc/usr/include/assert.h:

/opt/freescale/usr/local/gcc-4.4.78-eglibc-2.10.78-1/powerpc-linux-gnu/bin/../powerpc-linux-gnu/libc/usr/include/features.h:

/opt/freescale/usr/local/gcc-4.4.78-eglibc-2.10.78-1/powerpc-linux-gnu/bin/../powerpc-linux-gnu/libc/usr/include/bits/predefs.h:

/opt/freescale/usr/local/gcc-4.4.78-eglibc-2.10.78-1/powerpc-linux-gnu/bin/../powerpc-linux-gnu/libc/usr/include/sys/cdefs.h:

/opt/freescale/usr/local/gcc-4.4.78-eglibc-2.10.78-1/powerpc-linux-gnu/bin/../powerpc-linux-gnu/libc/usr/include/bits/wordsize.h:

/opt/freescale/usr/local/gcc-4.4.78-eglibc-2.10.78-1/powerpc-linux-gnu/bin/../powerpc-linux-gnu/libc/usr/include/gnu/stubs.h:

/opt/freescale/usr/local/gcc-4.4.78-eglibc-2.10.78-1/powerpc-linux-gnu/bin/../powerpc-linux-gnu/libc/usr/include/gnu/stubs-32.h:

/opt/freescale/usr/local/gcc-4.4.78-eglibc-2.10.78-1/powerpc-linux-gnu/bin/../lib/gcc/powerpc-linux-gnu/4.4.1/include/stddef.h:

../../../../mca_config.h:

/opt/freescale/usr/local/gcc-4.4.78-eglibc-2.10.78-1/powerpc-linux-gnu/bin/../powerpc-linux-gnu/libc/usr/include/inttypes.h:

/opt/freescale/usr/local/gcc-4.4.78-eglibc-2.10.78-1/powerpc-linux-gnu/bin/../powerpc-linux-gnu/libc/usr/include/stdint.h:

/opt/freescale/usr/local/gcc-4.4.78-eglibc-2.10.78-1/powerpc-linux-gnu/bin/../powerpc-linux-gnu/libc/usr/include/bits/wchar.h:

../../../../mca.h:

../../../../mca_impl_spec.h:

../../../../mrapi/src/mrapi_impl/mrapi_impl_spec.h:

/opt/freescale/usr/local/gcc-4.4.78-eglibc-2.10.78-1/powerpc-linux-gnu/bin/../powerpc-linux-gnu/libc/usr/include/signal.h:

/opt/freescale/usr/local/gcc-4.4.78-eglibc-2.10.78-1/powerpc-linux-gnu/bin/../powerpc-linux-gnu/libc/usr/include/bits/sigset.h:

/opt/freescale/usr/local/gcc-4.4.78-eglibc-2.10.78-1/powerpc-linux-gnu/bin/../powerpc-linux-gnu/libc/usr/include/bits/types.h:

/opt/freescale/usr/local/gcc-4.4.78-eglibc-2.10.78-1/powerpc-linux-gnu/bin/../powerpc-linux-gnu/libc/usr/include/bits/typesizes.h:

/opt/freescale/usr/local/gcc-4.4.78-eglibc-2.10.78-1/powerpc-linux-gnu/bin/../powerpc-linux-gnu/libc/usr/include/bits/signum.h:

/opt/freescale/usr/local/gcc-4.4.78-eglibc-2.10.78-1/powerpc-linux-gnu/bin/../powerpc-linux-gnu/libc/usr/include/time.h:

/opt/freescale/usr/local/gcc-4.4.78-eglibc-2.10.78-1/powerpc-linux-gnu/bin/../powerpc-linux-gnu/libc/usr/include/bits/siginfo.h:

/opt/freescale/usr/local/gcc-4.4.78-eglibc-2.10.78-1/powerpc-linux-gnu/bin/../powerpc-linux-gnu/libc/usr/include/bits/sigaction.h:

/opt/freescale/usr/local/gcc-4.4.78-eglibc-2.10.78-1/powerpc-linux-gnu/bin/../powerpc-linux-gnu/libc/usr/include/bits/sigcontext.h:

/opt/freescale/usr/local/gcc-4.4.78-eglibc-2.10.78-1/powerpc-linux-gnu/bin/../powerpc-linux-gnu/libc/usr/include/asm/sigcontext.h:

/opt/freescale/usr/local/gcc-4.4.78-eglibc-2.10.78-1/powerpc-linux-gnu/bin/../powerpc-linux-gnu/libc/usr/include/asm/ptrace.h:

/opt/freescale/usr/local/gcc-4.4.78-eglibc-2.10.78-1/powerpc-linux-gnu/bin/../powerpc-linux-gnu/libc/usr/include/bits/sigstack.h:

/opt/freescale/usr/local/gcc-4.4.78-eglibc-2.10.78-1/powerpc-linux-gnu/bin/../powerpc-linux-gnu/libc/usr/include/bits/pthreadtypes.h:

/opt/freescale/usr/local/gcc-4.4.78-eglibc-2.10.78-1/powerpc-linux-gnu/bin/../powerpc-linux-gnu/libc/usr/include/bits/sigthread.h:

/opt/freescale/usr/local/gcc-4.4.78-eglibc-2.10.78-1/powerpc-linux-gnu/bin/../powerpc-linux-gnu/libc/usr/include/pthread.h:

/opt/freescale/usr/local/gcc-4.4.78-eglibc-2.10.78-1/powerpc-linux-gnu/bin/../powerpc-linux-gnu/libc/usr/include/endian.h:

/opt/freescale/usr/local/gcc-4.4.78-eglibc-2.10.78-1/powerpc-linux-gnu/bin/../powerpc-linux-gnu/libc/usr/include/bits/endian.h:

/opt/freescale/usr/local/gcc-4.4.78-eglibc-2.10.78-1/powerpc-linux-gnu/bin/../powerpc-linux-gnu/libc/usr/include/bits/byteswap.h:

/opt/freescale/usr/local/gcc-4.4.78-eglibc-2.10.78-1/powerpc-linux-gnu/bin/../powerpc-linux-gnu/libc/usr/include/sched.h:

/opt/freescale/usr/local/gcc-4.4.78-eglibc-2.10.78-1/powerpc-linux-gnu/bin/../powerpc-linux-gnu/libc/usr/include/bits/sched.h:

/opt/freescale/usr/local/gcc-4.4.78-eglibc-2.10.78-1/powerpc-linux-gnu/bin/../powerpc-linux-gnu/libc/usr/include/bits/time.h:

/opt/freescale/usr/local/gcc-4.4.78-eglibc-2.10.78-1/powerpc-linux-gnu/bin/../powerpc-linux-gnu/libc/usr/include/xlocale.h:

/opt/freescale/usr/local/gcc-4.4.78-eglibc-2.10.78-1/powerpc-linux-gnu/bin/../powerpc-linux-gnu/libc/usr/include/bits/setjmp.h:

/opt/freescale/usr/local/gcc-4.4.78-eglibc-2.10.78-1/powerpc-linux-gnu/bin/../powerpc-linux-gnu/libc/usr/include/unistd.h:

/opt/freescale/usr/local/gcc-4.4.78-eglibc-2.10.78-1/powerpc-linux-gnu/bin/../powerpc-linux-gnu/libc/usr/include/bits/posix_opt.h:

/opt/freescale/usr/local/gcc-4.4.78-eglibc-2.10.78-1/powerpc-linux-gnu/bin/../powerpc-linux-gnu/libc/usr/include/bits/confname.h:

/opt/freescale/usr/local/gcc-4.4.78-eglibc-2.10.78-1/powerpc-linux-gnu/bin/../powerpc-linux-gnu/libc/usr/include/getopt.h:

/opt/freescale/usr/local/gcc-4.4.78-eglibc-2.10.78-1/powerpc-linux-gnu/bin/../powerpc-linux-gnu/libc/usr/include/sys/shm.h:

/opt/freescale/usr/local/gcc-4.4.78-eglibc-2.10.78-1/powerpc-linux-gnu/bin/../powerpc-linux-gnu/libc/usr/include/sys/ipc.h:

/opt/freescale/usr/local/gcc-4.4.78-eglibc-2.10.78-1/powerpc-linux-gnu/bin/../powerpc-linux-gnu/libc/usr/include/bits/ipctypes.h:

/opt/freescale/usr/local/gcc-4.4.78-eglibc-2.10.78-1/powerpc-linux-gnu/bin/../powerpc-linux-gnu/libc/usr/include/bits/ipc.h:

/opt/freescale/usr/local/gcc-4.4.78-eglibc-2.10.78-1/powerpc-linux-gnu/bin/../powerpc-linux-gnu/libc/usr/include/bits/shm.h:

/opt/freescale/usr/local/gcc-4.4.78-eglibc-2.10.78-1/powerpc-linux-gnu/bin/../powerpc-linux-gnu/libc/usr/include/sys/sem.h:

/opt/freescale/usr/local/gcc-4.4.78-eglibc-2.10.78-1/powerpc-linux-gnu/bin/../powerpc-linux-gnu/libc/usr/include/bits/sem.h:

/opt/freescale/usr/local/gcc-4.4.78-eglibc-2.10.78-1/powerpc-linux-gnu/bin/../powerpc-linux-gnu/libc/usr/include/sys/types.h:

/opt/freescale/usr/local/gcc-4.4.78-eglibc-2.10.78-1/powerpc-linux-gnu/bin/../powerpc-linux-gnu/libc/usr/include/sys/select.h:

/opt/freescale/usr/local/gcc-4.4.78-eglibc-2.10.78-1/powerpc-linux-gnu/bin/../powerpc-linux-gnu/libc/usr/include/bits/select.h:

/opt/freescale/usr/local/gcc-4.4.78-eglibc-2.10.78-1/powerpc-linux-gnu/bin/../powerpc-linux-gnu/libc/usr/include/sys/sysmacros.h:

/opt/freescale/usr/local/gcc-4.4.78-eglibc-2.10.78-1/powerpc-linux-gnu/bin/../powerpc-linux-gnu/libc/usr/include/ctype.h:

/opt/freescale/usr/local/gcc-4.4.78-eglibc-2.10.78-1/powerpc-linux-gnu/bin/../powerpc-linux-gnu/libc/usr/include/pwd.h:

/opt/freescale/usr/local/gcc-4.4.78-eglibc-2.10.78-1/powerpc-linux-gnu/bin/../powerpc-linux-gnu/libc/usr/include/stdio.h:

/opt/freescale/usr/local/gcc-4.4.78-eglibc-2.10.78-1/powerpc-linux-gnu/bin/../powerpc-linux-gnu/libc/usr/include/stdlib.h:

/opt/freescale/usr/local/gcc-4.4.78-eglibc-2.10.78-1/powerpc-linux-gnu/bin/../powerpc-linux-gnu/libc/usr/include/alloca.h:

/opt/freescale/usr/local/gcc-4.4.78-eglibc-2.10.78-1/powerpc-linux-gnu/bin/../powerpc-linux-gnu/libc/usr/include/string.h:

/opt/freescale/usr/local/gcc-4.4.78-eglibc-2.10.78-1/powerpc-linux-gnu/bin/../powerpc-linux-gnu/libc/usr/include/bits/string.h:

/opt/freescale/usr/local/gcc-4.4.78-eglibc-2.10.78-1/powerpc-linux-gnu/bin/../powerpc-linux-gnu/libc/usr/include/bits/string2.h:

/opt/freescale/usr/local/gcc-4.4.78-eglibc-2.10.78-1/powerpc-linux-gnu/bin/../powerpc-linux-gnu/libc/usr/include/errno.h:

/opt/freescale/usr/local/gcc-4.4.78-eglibc-2.10.78-1/powerpc-linux-gnu/bin/../powerpc-linux-gnu/libc/usr/include/bits/errno.h:

/opt/freescale/usr/local/gcc-4.4.78-eglibc-2.10.78-1/powerpc-linux-gnu/bin/../powerpc-linux-gnu/libc/usr/include/linux/errno.h:

/opt/freescale/usr/local/gcc-4.4.78-eglibc-2.10.78-1/powerpc-linux-gnu/bin/../powerpc-linux-gnu/libc/usr/include/asm/errno.h:

/opt/freescale/usr/local/gcc-4.4.78-eglibc-2.10.78-1/powerpc-linux-gnu/bin/../powerpc-linux-gnu/libc/usr/include/asm-generic/errno.h:

/opt/freescale/usr/local/gcc-4.4.78-eglibc-2.10.78-1/powerpc-linux-gnu/bin/../powerpc-linux-gnu/libc/usr/include/asm-generic/errno-base.h:

../../../../common/mca_utils.h:

/opt/freescale/usr/local/gcc-4.4.78-eglibc-2.10.78-1/powerpc-linux-gnu/bin/../lib/gcc/powerpc-linux-gnu/4.4.1/include/stdarg.h:

/opt/freescale/usr/local/gcc-4.4.78-eglibc-2.10.78-1/powerpc-linux-gnu/bin/../powerpc-linux-gnu/libc/usr/include/libio.h:

/opt/freescale/usr/local/gcc-4.4.78-eglibc-2.10.78-1/powerpc-linux-gnu/bin/../powerpc-linux-gnu/libc/usr/include/_G_config.h:

/opt/freescale/usr/local/gcc-4.4.78-eglibc-2.10.78-1/powerpc-linux-gnu/bin/../powerpc-linux-gnu/libc/usr/include/wchar.h:

/opt/freescale/usr/local/gcc-4.4.78-eglibc-2.10.78-1/powerpc-linux-gnu/bin/../powerpc-linux-gnu/libc/usr/include/bits/stdio_lim.h:

/opt/freescale/usr/local/gcc-4.4.78-eglibc-2.10.78-1/powerpc-linux-gnu/bin/../powerpc-linux-gnu/libc/usr/include/bits/sys_errlist.h:

/opt/freescale/usr/local/gcc-4.4.78-eglibc-2.10.78-1/powerpc-linux-gnu/bin/../powerpc-linux-gnu/libc/usr/include/bits/stdio.h:
