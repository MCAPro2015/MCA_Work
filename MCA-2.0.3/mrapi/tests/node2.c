/*
Copyright (c) 2010, The Multicore Association
All rights reserved.

Redistribution and use in source and binary forms, with or without
modification, are permitted provided that the following conditions are
met:

(1) Redistributions of source code must retain the above copyright
notice, this list of conditions and the following disclaimer.

(2) Redistributions in binary form must reproduce the above copyright
notice, this list of conditions and the following disclaimer in the
documentation and/or other materials provided with the distribution.

(3) Neither the name of the Multicore Association nor the names of its
contributors may be used to endorse or promote products derived from
this software without specific prior written permission.

THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS
IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED
TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A
PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER
OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL,
EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO,
PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR
PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF
LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING
NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.

*/
/*
This test makes sure that calling initialize for the same node/domain but from a 
different pid/tid will error out
*/

#include <mrapi.h>

char status_buff[MRAPI_MAX_STATUS_SIZE];

char status_buff[MRAPI_MAX_STATUS_SIZE];
#include <stdio.h>
#include <stdlib.h>
#include <pthread.h>
#include <string.h>
#include <unistd.h>

#ifndef NODE
#define NODE 1
#endif

#ifndef DOMAIN
#define DOMAIN 1
#endif

#define LOCK_LIMIT 5

#define NUM_THREADS 1
#define WRONG(x) wrong(x,__LINE__);

void wrong(mca_status_t status,unsigned line)
{
  fprintf(stderr,"WRONG: line=%u status=%s",line,mrapi_display_status(status,status_buff,sizeof(status_buff)));
  fflush(stdout);
  exit(1);
}

typedef struct {
  mca_node_t tid;
} thread_data;

int x;

void *run_thread(void* tdata) {
  mca_status_t status;
  mrapi_info_t version;
  mrapi_parameters_t parms = 0;

  thread_data* t = (thread_data*)tdata;
  printf("run_thread...");
  fflush(stdout);
  mrapi_initialize(DOMAIN,NODE+t->tid,parms,&version,&status);
  if (status == MRAPI_SUCCESS) {  WRONG(status) }

  return NULL;
}


int main () {
  int rc = 0;
  int i = 0;
  pthread_t threads[NUM_THREADS];
  thread_data tdata[NUM_THREADS];
  mrapi_status_t status;
  mrapi_info_t version;
  mrapi_parameters_t parms = 0;
  mrapi_set_debug_level(1);


  tdata[0].tid = 0;

  printf("main.....");
  mrapi_initialize(DOMAIN,NODE,parms,&version,&status);
  if (status != MRAPI_SUCCESS) { WRONG(status) }
  fflush(stdout);

  rc += pthread_create(&threads[0], NULL, run_thread,(void*)&tdata[0]);

  
  for (i = 0; i < NUM_THREADS; i++) {
    pthread_join(threads[i],NULL);
  }
  
  mrapi_finalize(&status);
  if (status != MRAPI_SUCCESS) { WRONG(status) }
  
  if (rc == 0) {
    printf("   Test PASSED\n");
  }
  
  return rc;
}
