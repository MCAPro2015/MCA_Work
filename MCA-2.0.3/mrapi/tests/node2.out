ERROR mca_set_debug_level : This library was built without debug support.
  If you want to enable debugging, re-build with the --enable-debug option.
main.....run_thread...   Test PASSED
